telemetry/telescope_ec2_age/instance_uptime.py

```diff
@@ -34,10 +34,25 @@ def describe_instances(instance_list):
                 launch_time = instance["LaunchTime"]
                 instance_id_name = instance["InstanceId"]
                 instance_launch_times[instance_id_name] = launch_time
+
+                logger.debug(instance_id_name)
+                logger.debug(launch_time)
+        logger.info("Described with no invalid IDs")
         return instance_launch_times
+
     except ClientError as e:
-        logger.error(str(e))
-        return None
+        instance_launch_times = {}
+        for row in instance_list:
```

Comment:
I would call this var `instance` instead of `row`. Makes it easier IMO to read within the `for` loop without having to look at what the abstract var `row` refers to.